pyccel/stdlib/ndarrays/ndarrays.h

```diff
@@ -9,6 +9,7 @@
 # include <complex.h>
 # include <stdbool.h>
 # include <stdint.h>
+# include <math.h>
```

Comment:
Why does `ndarrays.h` need a new dependence if you haven't modified this file?

Reply:
https://github.com/pyccel/pyccel/pull/1169/commits/5f170614f3bdfc8fa25ebc2e2a752d282dca8896